plugin/build.gradle

```diff
@@ -298,7 +298,9 @@ List<String> jacocoExclusions = [
         'org.opensearch.ml.cluster.MLSyncUpCron',
         'org.opensearch.ml.model.MLModelGroupManager',
         'org.opensearch.ml.helper.ModelAccessControlHelper',
-        'org.opensearch.ml.action.models.DeleteModelTransportAction.2'
+        'org.opensearch.ml.action.models.DeleteModelTransportAction.2',
```

Comment:
This seems blocking several other PRs which merging to main branch. Can you split this part to a new PR ? So we can merge to main branch and unblock others quickly